packages/circuit-ui/components/ImageInput/ImageInput.docs.mdx

```diff
@@ -49,3 +49,17 @@ There are some caveats with this approach:
 ### Using a div element
 
 <Story id="forms-imageinput--custom-component-div" />
+
+## Accessibility
+
+### Best practices
+
+#### Error message associated with input field
+
+Error messages should be simple and clear to understand, also provide guidance on how to correct mistakes.
+The error messages must be associated with their respective controls (e.g. Input field), via labeling or `aria-describedby`.
+The screen readers call the error message with a live region by applying an `aria-live` property or using one of the live region roles (e.g alert).
+
+The `aria-describedby` is added to the Input field, and `aria-invalid="true"` when the error is triggered.
+The `aria-live="assertive"` indicate that screen readers should immediately announce the error message rather than completing other announcements first.
```

Comment:
What source did you find for this? Why did you start by choosing polite first? I can also see that it could be pretty disruptive. Whatever we choose, it would be great to back it up

Reply:
Will update docs more today and add the resources.

Reply:
Just a question, I added an article to the different techniques they did and testing. Should I also note down the current known issues with VoiceOver and iframes?